examples/test_data_gen/test_data_gen_local/run_test_data_gen.py

```diff
@@ -0,0 +1,116 @@
+import json
+import os
+from datetime import datetime
+
+import configargparse
+from constants import TEXT_CHUNK
+from doc_split import split_doc
+
+from promptflow import PFClient
+from promptflow.entities import Run
+
+CONFIG_FILE = os.path.abspath(os.path.join(os.path.dirname(__file__), "config.ini"))
+
+
+def batch_run_flow(
+    pf: PFClient,
+    flow_folder: str,
+    flow_input_data: str,
+    flow_batch_run_size: int,
+    connection_name: str = "azure_open_ai_connection",
+):
+    environment_variables = {
+        "PF_WORKER_COUNT": str(flow_batch_run_size),
+        "PF_BATCH_METHOD": "spawn",
+    }  # TODO: what does 'spawn' mean?
+
+    print("start to run batch flow run.")
+    # create run
+    base_run = pf.run(
+        flow=flow_folder,
+        data=flow_input_data,
+        stream=True,  # TODO: understand 'stream'
+        environment_variables=environment_variables,
+        connections={
+            "validate_and_generate_seed_question": {"connection": connection_name},
+            "validate_and_generate_test_question": {"connection": connection_name},
+            "validate_and_generate_context": {"connection": connection_name},
+            "generate_answer": {"connection": connection_name},
+        },
+        column_mapping={TEXT_CHUNK: "${data.text_chunk}"},
+        debug=True,
+    )
+
+    return base_run
+
+
+def get_batch_run_output(pf: PFClient, base_run: Run):
+    print(f"Start to get batch run {base_run} details.")
```

Comment:
Print {base_run} would output a big run object body. I think you only want to print the run id here.

Reply:
In a one-line logging, print name would be good enough.

print(f"Start to get batch run {base_run.name} details.")